packages/docs/pages/multi-select.tsx

```diff
@@ -1,397 +1,483 @@
+                  <CodePreview>
+                    {/* jsx-to-string:start */}
+                    {function Example() {
+                      const [value, setValue] = useState(['mx']);
+                      const handleChange = (val) => setValue(val);
+
+                      return (
+                        <Form>
+                          <FormGroup>
+                            <MultiSelect
+                              action={{
+                                actionType: 'destructive' as const,
+                                content: 'Remove Country',
+                                icon: <DeleteIcon />,
+                                onActionClick: () => null,
+                              }}
+                              filterable={true}
+                              label="Countries"
+                              maxHeight={300}
+                              onOptionsChange={handleChange}
+                              options={[
+                                { value: 'us', content: 'United States' },
+                                { value: 'mx', content: 'Mexico' },
+                                { value: 'ca', content: 'Canada' },
+                                { value: 'en', content: 'England' },
+                                { value: 'fr', content: 'France' },
+                                { value: 'gr', content: 'Germany' },
+                                { value: 'ar', content: 'Argentina' },
+                                { value: 'ru', content: 'Russia', disabled: true },
+                                { value: 'ch', content: 'Chile' },
+                                { value: 'bo', content: 'Bolivia' },
+                                { value: 'jp', content: 'Japan' },
+                                { value: 'cn', content: 'China' },
+                                { value: 'sk', content: 'South Korea' },
+                                { value: 'au', content: 'Australia' },
+                                { value: 'ug', content: 'Uganda' },
+                              ]}
+                              placeholder={'Choose country'}
+                              placement={'bottom-start'}
+                              required
+                              value={value}
+                            />
+                          </FormGroup>
+                        </Form>
+                      );
+                    }}
+                    {/* jsx-to-string:end */}
+                  </CodePreview>
+                </Fragment>
+              ),
+            },
+            {
+              id: 'position',
+              title: 'Position',
+              render: () => (
+                <Fragment key="position">
+                  <Text>
+                    <Code primary>MultiSelect</Code> can be anchored in different directions with the{' '}
+                    <Code primary>placement</Code> property. It will automatically find a position if there's not enough
```

Comment:
I updated ` <Code primary>position</Code>` to `  <Code primary>placement</Code>` since I think this implementation is talking about different ways of using the placement property. Example:
` placement="top"`
`placement="bottom-start"`
...and so on.